stacker/config/__init__.py

```diff
@@ -103,6 +113,24 @@ def parse(raw_config):
 
     """
 
+    # Convert any applicable dictionaries back into lists
+    config_dict = yaml.safe_load(raw_config)
```

Comment:
Ok, rather than using the default safe_load, lets do something that turns the YAML dictionaries into `OrderedDicts` in python.  I found and tested this method, and it seems like it works: https://stackoverflow.com/a/21912744/3692133